torchvision/transforms/v2/functional/_utils.py

```diff
@@ -100,21 +100,14 @@ def _get_kernel(functional, input_type, *, allow_passthrough=False):
     if not registry:
         raise ValueError(f"No kernel registered for functional {functional.__name__}.")
 
-    # In case we have an exact type match, we take a shortcut.
-    if input_type in registry:
-        return registry[input_type]
```

Comment:
why remove this?
I understand it's not strictly needed, but it wasn't strictly needed before, so why remove it now?